src/jdk.internal.vm.compiler/share/classes/org.graalvm.compiler.hotspot/src/org/graalvm/compiler/hotspot/HotSpotHostBackend.java

```diff
@@ -130,39 +130,37 @@ protected CallingConvention makeCallingConvention(StructuredGraph graph, Stub st
     }
 
     public void emitStackOverflowCheck(CompilationResultBuilder crb) {
-        if (config.useStackBanging) {
```

Comment:
I think you can leave this file unchanged with my suggested modification in GraalHotSpotVMConfig.java.

Reply:
done